elytron/WFCORE-4496-principal-transformer-in-aggregate-realm.adoc

```diff
@@ -0,0 +1,67 @@
+= [WFCORE-4496] Adding principal-transformer in aggregate-realm
+:author:            Ashley Abdel-Sayed
+:email:             aabdelsa@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+Elytron currently has an aggregate security realm which is a combination of two or more realms; an authentication realm
+and one or more authorization realms. There is currently no way to transform the principal in between loading the authentication
+identity and loading the authorization identity. This enhancement would allow transformations to be applied to the principal
+after loading the authentication identity and before loading the authorization identity.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFCORE-4496[WFCORE-4496]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1220[EAP7-1220]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+* mailto:vmarek@redhat.com[Vratislav Marek]
+
+=== Testing By
+[ ] Engineering
+
+[x] QE
+
+=== Affected Projects or Components
+* WildFly, Security
```

Comment:
Missing projects Wildfly-core and Elytron.

Reply:
Added